rootfs/etc/nginx/template/nginx.tmpl

```diff
@@ -487,10 +487,31 @@ http {
         set $proxy_upstream_name "-";
 
         location {{ $healthzURI }} {
+            {{ if $cfg.EnableOpentracing }}
+            opentracing off;
+            {{ end }}
             access_log off;
             return 200;
         }
 
+        location /is-dynamic-lb-initialized {
+            {{ if $cfg.EnableOpentracing }}
+            opentracing off;
+            {{ end }}
+            access_log off;
+
+            content_by_lua_block {
+                local backend_data = ngx.shared.configuration_data:get("backends")
```

Comment:
can you use the helper function instead? https://github.com/kubernetes/ingress-nginx/blob/master/rootfs/etc/nginx/lua/configuration.lua#L6

Reply:
done